example-3/README.md

```diff
@@ -0,0 +1,42 @@
+# Examples 3
+
+For more or other examples please check out our main [README](../)
+
+This example showcases
+
+- Round inputs
+- Native CSS styling
+- ES5 Compatible
+
+[HTML](example-3/index.html) | [CSS](example-3/style.css) | [Javascript](example-3/script.js)
+
+![](screenshots/example-3.gif)
```

Comment:
There appears to be no screenshots. You can add one or remove this line instead

Reply:
Screenshots will be added later, we'll ask Jaap.